src/cluster.cc

```diff
@@ -564,10 +566,11 @@ Status Cluster::ParseClusterNodes(const std::string &nodes_str, ClusterNodes *no
       int start, stop;
       std::vector<std::string> ranges = Util::Split(fields[i], "-");
       if (ranges.size() == 1) {
-        start = std::atoi(ranges[0].c_str());
-        if (IsValidSlot(start) == false) {
+        auto parse_result = ParseInt<int>(ranges[0].c_str(), NumericRange<int>{0, kClusterSlots - 1}, 10);
```

Comment:
Declare `NumericRange<int>{0, kClusterSlots - 1}` as a variable? It appears 3 times in this function.

Reply:
Thanks, great advice.